pyeapi/client.py

```diff
@@ -395,14 +395,27 @@ def connect(transport=None, host='localhost', username='admin',
         port (int): The TCP port of the endpoint for the eAPI connection.  If
             this keyword is not specified, the default value is automatically
             determined by the transport type. (http=80, https=443)
+        return_node (bool): Returns a Node object if True, otherwise
+            returns an EapiConnection object.
+
 
     Returns:
-        A instance of an EapiConnection object for the specified transport.
+        An instance of an EapiConnection object for the specified transport.
 
     """
     transport = transport or DEFAULT_TRANSPORT
-    return make_connection(transport, host=host, username=username,
+    connection = make_connection(transport, host=host, username=username,
                            password=password, port=port, timeout=timeout)
+    if return_node:
+        kwargs = dict(
+            transport=transport,
+            host=host,
+            username=username,
+            password=password,
+            port=port,
+        )
+        return Node(connection, **kwargs)
```

Comment:
Why build a `kwargs` dict? Can't you just return a `Node(connection, transport=transport, etc...)`?